docs/upgrading.md

```diff
@@ -11,6 +11,7 @@ Multiple backwards incompatible changes:
 
 - The service config no longer accepts the `network` key on the first YAML page. The network can now be defined via the package overrides.
 - Dependency specifications are now checked against imports for all packages, this means initially packages might need modifying to reference/unreference missing/irrelevant dependencies.
+- The global configuration file for the `aea`/`autonomy` CLI has a breaking change. Please remove `~/.aea/cli_config.yaml` and rerun `autonomy init --remote`.
```

Comment:
@angrybayblade you need to describe what commands changed between `0.1.6` and `0.2.0`!

Reply:
I'll do this in the release PR

Reply:
Ok - in release PR both for the previous and current changes :)